templates/bet/Inspector.tsx

```
@@ -0,0 +1,257 @@
+    const claimInputRef = useRef<HTMLInputElement>(null)
+    const opponentInputRef = useRef<HTMLInputElement>(null)
+    const arbitratorInputRef = useRef<HTMLInputElement>(null)
+    const assetInputRef = useRef<HTMLInputElement>(null)
+    const amountInputRef = useRef<HTMLInputElement>(null)
+
+    const fid = useFarcasterId()
+    const username = useFarcasterName()
+    const uploadImage = useUploadImage()
+
+    // Set owner if it's not already set
+    useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: updateConfig

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)